application/src/Weekend.php

```diff
@@ -4,49 +4,77 @@
 
 class Weekend
 {
+    const MAPPING = [
+        '\o/'  => '🎉',
+        '◄:•D' => '🥳',
+        'B-)'  => '😎',
+        '¬‿¬'  => '😏',
+        ':('   => '😟',
+        'X-('  => '😠',
+    ];
+
+    /**
+     * Compute the main text with emojis
+     *
+     * @return string
+     */
+    public function getRichText()
```

Comment:
```suggestion
    /**
     * Compute the main text with emojis
     */
    public function getRichText(): string
```

Reply:
Thanks, I've done this on other methods: `getText()` and `getSubText()`.